src/modules/TasksModule/components/TaskesData/TaskesData.tsx

```diff
@@ -1,10 +1,220 @@
-
+import axios from "axios";
+import { SetStateAction, useEffect, useState } from "react";
+import { useNavigate ,Link} from "react-router-dom";
+import { toast } from "react-toastify";
+import { SubmitHandler, useForm } from "react-hook-form";
+import { baseUrl } from "../../../../Context/baseUrl";
+import Select from "react-select";
+interface FormValues {
+  title: string;
+  description: string;
+  employeeId?: number;
+  projectId?: number;
+}
+interface projectType {
+  id: number;
+  title: string;
+  description: string;
+}
 const TaskesData = () => {
+  const [userList, setUserList] = useState([]);
+  const [projectList, setProjectList] = useState([]);
+  const [selectedUser, setSelectedUser] = useState(null);
+  const [isLoading, setIsLoading] = useState(false);
+
+  const navigate = useNavigate();
+
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+    setValue,
+    setError,
+  } = useForm<FormValues>();
+  const goBack = () => {
+    navigate(-1);
+  };
+  // Fucntion to add new tasks
+  const addNewTask: SubmitHandler<FormValues> = async (data: FormValues) => {
+    try {
+      const response = await axios.post(`${baseUrl}/Task`, data, {});
+      console.log(response);
```

Comment:
We need to remove logs to avoid revealing sensitive data. We need to also provide a fallback message in case there was no message coming from the backend.